langchain4j/src/test/java/dev/langchain4j/service/StreamingAiServicesWithToolsIT.java

```diff
@@ -130,5 +130,36 @@ void should_use_tool_with_List_of_Strings_parameter(StreamingChatLanguageModel m
         );
     }
 
+    @ParameterizedTest
+    @MethodSource("models")
+    void should_use_tool_without_memory(StreamingChatLanguageModel model) throws Exception {
```

Comment:
Could you please follow the same approach as in `AiServicesWithToolsWithoutMemoryIT`? You could create `StreamingAiServicesWithToolsWithoutMemoryIT` and add all 3 test methods there

Reply:
No problem ~ I will create a `StreamingAiServicesWithToolsWithoutMemoryIT` test class